e2e/test/scenarios/admin/database-routing/database-routing-admin.cy.spec.ts

```diff
@@ -457,11 +457,13 @@ describe("admin > database > database routing", () => {
           cy.findByText("Remove").should("not.exist");
           cy.findByText("Edit").should("exist").click();
         });
-        H.modal().within(() => {
-          H.typeAndBlurUsingLabel(/Slug/, "Destination DB 1");
-          cy.button("Save changes").click();
-          cy.wait("@databaseUpdate");
-        });
+        // This is currently broken with user denied permission
```

Comment:
Is there a way to mark the test case as "skipped" instead of commenting out?

Reply:
Oh, I'll uncomment this. It'll be fixed after a rebase. One sec.